newm-server/src/main/kotlin/io/newm/server/features/song/database/ReleaseEntity.kt

```diff
@@ -36,7 +36,7 @@ class ReleaseEntity(id: EntityID<ReleaseId>) : UUIDEntity(id) {
     var arweaveCoverArtUrl: String? by ReleaseTable.arweaveCoverArtUrl
     var hasSubmittedForDistribution: Boolean by ReleaseTable.hasSubmittedForDistribution
     var errorMessage: String? by ReleaseTable.errorMessage
-    var forceDistributed: Boolean by ReleaseTable.forceDistributed
+    var forceDistributed: Boolean? by ReleaseTable.forceDistributed
```

Comment:
Do we need 3 states for this value? True / False / null ?

Reply:
It previously lived in the song table and was a nullable value since it's @Transient. We never send this value down to the web client. By keeping it the same as it was before, we prevent it from interfering with our integration tests when we compare expected to actual values.